.env

```diff
@@ -65,6 +65,11 @@ CONCEPT_COMPLEX_UUID=
 OPENMRS_IDENTIFIER_TYPE_UUD=
 DB_EVENT_DESTINATIONS=direct:odoo-event-listener
 CREATE_CUSTOMER_IF_NOT_EXIST=false
+ODOO_CUSTOMER_EXTERNAL_ID=x_external_identifier
+ODOO_CUSTOMER_DOB_FIELD=x_customer_dob
+ODOO_CUSTOMER_WEIGHT_FIELD=x_customer_weight
+EMR_WEIGHT_CONCEPT=5089AAAAAAAAAAAAAAAAAAAAAAAAAAAA
+ODOO_ENABLE_EXTRA_CUSTOMER_FIELDS_ROUTE=true
```

Comment:
Could we name this var differently? Maybe describing what the said route does.
For instance:
`SYNC_ADDITIONAL_CUSTOMER_FIELDS`